upgrading/upgrade-helm.adoc

```diff
@@ -15,7 +15,7 @@ You must follow a specific upgrade path for {product-title-short} depending on t
 When upgrading from earlier releases, follow this guidance:
 
 * If the release for Central is earlier than 3.74, you must upgrade to the latest 3.74 patch before upgrading to a 4.x release. See the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/3.74/html/upgrading/index[upgrade documentation for version 3.74] for information about upgrades from earlier versions to 3.74.
```

Comment:
```suggestion
* If the release for Central is earlier than 3.74, you must upgrade to the latest 3.74 patch before upgrading to a 4.x release. See the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/3.74/html/upgrading/index[upgrade documentation for version 3.74] for information about upgrades from earlier versions through 3.74.
```

Reply:
After thinking about it, I think "to" is better than "through" since the documentation being referred to describes upgrading to 3.74 and not any other versions before 3.74.... Basically, everyone needs to upgrade to a minimum of 3.74 before then upgrading to 4.0+ (there is no 3.75). I hope that makes sense.

Reply:
Thanks, I do appreciate the suggestions and the peer review though!